shared/rakefiles/xk_infra.rake

```diff
@@ -88,7 +88,14 @@ task :apply_common_infra => [@gcp_creds_file] do
     #{@exekube_cmd} gcloud organizations get-iam-policy #{ENV["ORGANIZATION_ID"]} --format=json
   }
   permissions_list = JSON.parse(permissions_list_json)["bindings"]
-  organizations_permissions = ["roles/resourcemanager.projectCreator", "roles/billing.user"]
+  organizations_permissions = ["roles/billing.projectManager",
+                               "roles/billing.user",
+                               "roles/iam.organizationRoleViewer",
+                               "roles/iam.serviceAccountAdmin",
+                               "roles/iam.serviceAccountKeyAdmin",
+                               "roles/resourcemanager.projectCreator",
+                               "roles/resourcemanager.projectIamAdmin",
+                               "roles/serviceusage.serviceUsageAdmin"]
```

Comment:
Same, are all of the above necessary on org level?

Reply:
The SA needs to enable "iam.googleapis.com" service before setting the IAMs at the project level.